src/config/subnet.rs

```diff
@@ -1,10 +1,13 @@
+use std::collections::HashMap;
 use ipc_sdk::subnet_id::SubnetID;
 use url::Url;
 use fvm_shared::address::Address;
 use serde::Deserialize;
 
 use crate::config::deserialize::{deserialize_accounts, deserialize_subnet_id};
 
+pub type Subnets = HashMap<String, Subnet>;
```

Comment:
nit: I know you already merged this, but I feel like defining a type for this obscures the code a bit as it adds a layer of opacity to the underlying type.